src/Microsoft.Build.Tasks.Git/buildTransitive/Microsoft.Build.Tasks.Git.props

```diff
@@ -0,0 +1,4 @@
+<!-- Licensed to the .NET Foundation under one or more agreements. The .NET Foundation licenses this file to you under the MIT license. See the License.txt file in the project root for more information. -->
+<Project>
+  <Import Project="..\build\$(MSBuildThisFileName).props"/>
```

Comment:
>Import [](http://example.com/codeflow?start=3&length=6)

Why do we need buildTransitive? Seems like a project that depends on a project that has Source Link may or may not opt-into having Source Link. Either way it should work as there is no cross-project dependency on source link.

Reply:
buildTransitive is necessary so that packages that reference Microsoft.Build.Tasks.Git.nupkg receive the msbuild props and targets files from it transitively. That's necessary when using NuGet's Pack task as it emits a slightly different nuspec as when hand-authoring it.

https://github.com/NuGet/Home/issues/6091 has all the details.